README.md

```diff
@@ -29,7 +29,21 @@ Install tkinter to be able to display graphs
 `sudo apt install python3-tk`
 
 ### Windows  
-Help wanted  
+Make sure pip for python 3 is installed  
+`pip3 --version` should not print an error  
+  
+Install pipenv via pip  
+`pip install pipenv`  
+
+Clone the repo and extract it
+
+Install the project dependencies  
+`cd Probability-Programs`  
+`pipenv install`  
+Make sure you've the `"%AppData%\Python\Python3X\Scripts"` directory added to PATH env variable.
```

Comment:
change to
`Make sure the `"%AppData%\Python\Python3X\Scripts"` directory is added to PATH env variable.`